lib/business/data/betalingsservice.yml

```diff
@@ -15,6 +15,31 @@ holidays:
   - May 25th, 2017
   - May 26th, 2017
   - June 5th, 2017
-  - December 24th, 2017
```

Comment:
24th December 2017 is a Sunday which is not a working day (at the top of this file) so this entry is redundant.